test_contribute.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+import contribute
+from subprocess import check_output
+import os
+
+class TestContribute(unittest.TestCase):
+
+
+    def test_arguments(self):
+        args = contribute.arguments(['-nw'])
+        self.assertTrue(args.no_weekends)
+        self.assertEqual(args.max_commits,10)
+        self.assertTrue(1 <= contribute.contributions_per_day(args) <= 20)
+    
+    def test_contributions_per_day(self):
+        args = contribute.arguments(['-nw'])
+        self.assertTrue(1 <= contribute.contributions_per_day(args) <= 20)
+
+    def test_commits(self):
+        contribute.NUM = 11 #limiting the number only for unittesting
+        contribute.main(['-nw','--user_name=sampleusername','--user_email=your-username@users.noreply.github.com'])
```

Comment:
Could you please explicitly specify all cli options (except the repository option). This way the test would also validate that the cli options work correctly.